js/main.js

```diff
@@ -0,0 +1,33 @@
+var $photoUrlInput = document.querySelector('#photoUrl');
+var $photo = document.querySelector('.photo');
+var $textInputs = document.querySelectorAll('.text-adjust');
+
+$photoUrlInput.addEventListener('input', changeUrl);
+
+function changeUrl(event) {
+  $photo.src = event.target.value;
+}
+
+var $formElement = document.querySelector('form');
+$formElement.addEventListener('submit', formSubmitted);
+
+function formSubmitted(event) {
+  event.preventDefault();
+  var newEntry = {
+    title: $textInputs[0].value,
+    photoUrl: $textInputs[1].value,
+    comment: $textInputs[2].value,
+    EntryId: data.nextEntryId
```

Comment:
Property names should not begin with a capital letter, you should name this `entryId`